internal/otelcollector/config/otlpexporter/config_builder.go

```diff
@@ -74,6 +75,32 @@ func ExporterID(output *telemetryv1alpha1.OtlpOutput, pipelineName string) strin
 	return fmt.Sprintf("%s/%s", outputType, pipelineName)
 }
 
+func makeTLSConfig(output *telemetryv1alpha1.OtlpOutput, otlpEndpointValue, pipelineName string) config.TLS {
+	var cfg config.TLS
+	cfg.Insecure = isInsecureOutput(otlpEndpointValue)
+
+	if output.TLS == nil {
+		return cfg
+	}
+
+	if !cfg.Insecure {
```

Comment:
I restored the isInsecureOutput function that checks if the endpoint scheme is http. It is necessary that this is set before line 82 (`output.TLS == nil`) to have the same functionality as before. 

This line checks if the output is not insecure, i.e., line 80 (`cfg.Insecure = isInsecureOutput(otlpEndpointValue)`) returned false. If this is the case, we allow to overwrite the TLS setting.